app/serializers/api/product_serializer.rb

```diff
@@ -32,18 +31,6 @@ def variants
     options[:variants][object.id] || []
   end
 
-  def master
-    options[:master_variants][object.id].andand.first
-  end
-
-  def price
-    if options[:enterprise_fee_calculator]
-      object.master.price + options[:enterprise_fee_calculator].indexed_fees_for(object.master)
```

Comment:
This is the line where https://app.bugsnag.com/yaycode/openfoodnetwork-uk/errors/60adfcc31cf9740007f82fc9 fails.

Reply:
Hmmm yeah that looks really suspect. I don't think we can ever apply fees to a master variant!